build_tools/scripts/generate_compilation_flagfile.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Generates a compilation flagfile for iree-compiler."""
```

Comment:
I think it would be good to comment why this is necessary (CMake is stupid)

Reply:
Done.